packages/client/components/TransactionList.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Text, Box } from "native-base";
+import { StyleSheet } from "react-native";
+import { Transaction as TransactionEntity } from "@cryptify/common/src/domain/entities/transaction";
+import { Transaction } from "./Transaction";
```

Comment:
Rename this file to a better name that doesnt conflict with the entity

```suggestion
import { TransactionListItem } from "./TransactionListItem";
```